src/Tribe/Aggregator/Record/CSV.php

```diff
@@ -193,6 +193,37 @@ public function get_content_type() {
 	}
 
 	/**
+	 * Translates the posttype-driven content types to content types that the CSV importer knows
+	 *
+	 * @param string $content_type Content Type
+	 *
+	 * @return string CSV Importer compatible content type
+	 */
+	public function get_csv_content_type( $content_type = null ) {
+
+		if ( ! $content_type ) {
+			$content_type = $this->get_content_type();
+		}
+
+		switch ( $content_type ) {
```

Comment:
I've tweaked the above slightly to account for the fact that some values passed in (or fetched from `get_content_type()`) are what is actually desired.